dotnet-desktop-guide/net/winforms/data/snippets/how-to-synchronize-multiple-controls/vb/Form1.vb

```diff
@@ -0,0 +1,47 @@
+﻿'<bind_controls_using_BindingSource>
```

Comment:
This should be moved down. Currently the `Class` declaration is visible but the `End Class` isn't.

Reply:
ok